api/src/public/login/script.js

```diff
@@ -234,17 +239,25 @@ const checkUnsupportedBrowser = () => {
     document.getElementById('unsupported-browser-update').setAttribute('translate', outdatedComponentKey);
     document.getElementById('unsupported-browser-update').innerText =
       translations[selectedLocale][outdatedComponentKey];
-    document.getElementById('unsupported-browser').classList.remove('hidden');
+    document.getElementById('unsupported-browser').classList.remove(HIDDEN_CLASS);
   }
 };
 
+const togglePassword = () => {
+  const input = document.getElementById(PASSWORD_INPUT_ID);
+  input.type = input.type === 'password' ? 'text' : 'password';
+  document.getElementById('show-password').classList.toggle(HIDDEN_CLASS);
+  document.getElementById('hide-password').classList.toggle(HIDDEN_CLASS);
```

Comment:
I think the code comes out slightly cleaner if you toggle a class on a parent, for example, adding a class to the `#password-container` element. If you name the class something sensible like "mask" (or a better name) then you can change the style of both the icon elements with one line of JS.

Reply:
Okay. I was thinking that, or keeping the CSS minimal and reusing the classes we already had